Sources/MapboxMaps/Style/Style.swift

```diff
@@ -930,6 +930,74 @@ public final class Style: StyleProtocol {
         return styleManager.getStyleTerrainProperty(forProperty: property)
     }
 
+    // MARK: - Atmosphere
+
+    /// Set the atmosphere of the style
+    /// - Parameter atmosphere: ``Atmosphere`` object describing the fog, space and stars.
+    public func setAtmosphere(_ atmosphere: Atmosphere) throws {
+        guard let atmosphereDictionary = try atmosphere.toJSON() as? [String: Any] else {
+            throw TypeConversionError.unexpectedType
+        }
+
+        try setAtmosphere(properties: atmosphereDictionary)
+    }
+
+    /// Remove the atmosphere of the style. No fog, space or stars would be rendered.
+    public func removeAtmosphere() throws {
+        styleManager.setStyleAtmosphereForProperties(NSNull())
```

Comment:
Should this call be wrapped with `try handleExpected`?

Reply:
Good catch!